addons/mail/static/src/discuss/gif_picker/common/gif_picker.xml

```diff
@@ -1,6 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <templates xml:space="preserve">
     <t t-name="discuss.GifPicker">
+        <t t-if="store?.exists()" t-call="discuss.GifPicker.main"/>
+    </t>
+
+    <t t-name="discuss.GifPicker.main">
```

Comment:
Globally used this pattern to guard `store.exists()` as template could re-render when records have been destroyed. All this diff could be avoided if we find a way to destroy the records after the full cleanup of app mount on test fixture.

Reply:
Why not just have a top level t-if instead of creating a new template?
This can't be done in stable as it could break overrides targeting the main table.